merge-coverage.ts

```diff
@@ -0,0 +1,38 @@
+import * as fs from 'fs-extra';
+import { createReporter } from 'istanbul-api';
+import { createCoverageMap } from 'istanbul-lib-coverage';
+import * as yargs from 'yargs';
+
+main().catch((err) => {
```

Comment:
Codacy found an issue: ['main' was used before it was defined.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5767244)